spec/spec_helper.rb

```diff
@@ -32,6 +32,10 @@ def get_plugin(plugin, ohai = Ohai::System.new, path = PLUGIN_PATH)
   loader.load_plugin(File.join(path, "#{plugin}.rb"))
 end
 
+def convert_window_output(stdout)
```

Comment:
I think @btm is just saying this should be "windows" with an "s" at the end, minor typo.